integration/core/test_api.py

```diff
@@ -82,3 +82,16 @@ def test_template_base(client):
     assert len(templates) > 0
     for i in range(len(templates)):
         assert templates[i].templateBase is not None
+
+
+def test_template_many_versions(client):
+    templates = client.list_template(catalogId='qa-catalog')
+    assert len(templates) > 0
+
+    for i in range(len(templates)):
+        if templates[i].id == unicode('qa-catalog:many-versions'):
+            versionUrls = templates[i].versionLinks.values()
+            for i in range(len(versionUrls)):
+                version_response = requests.get(versionUrls[i])
+                assert version_response is not 404
+                assert "/docker-compose.yml" not in version_response.text
```

Comment:
requests has a method `version_response.json()` where it will parse it as json.  Then you can assert that the field is set to what it is supposed to be.  It would be better if you assert that the value equals what you expect, not the opposite.